src/main/java/rs117/hd/data/materials/Material.java

```diff
@@ -334,6 +337,30 @@
 
 	WALL_STONE_N,
 	WALL_STONE(p -> p.setNormalMap(WALL_STONE_N)),
+	METALLIC_1,
+	METALLIC_1_SEMIGLOSS(METALLIC_1, p -> p
+			.setSpecular(0.3f, 80)
+	),
+	METALLIC_1_GLOSS(METALLIC_1, p -> p
+			.setSpecular(0.7f, 80)
+	),
+	METALLIC_1_HIGHGLOSS(METALLIC_1, p -> p
+			.setSpecular(1.1f, 80)
+	),
+	METALLIC_2,
```

Comment:
This is exactly what you are asking for, the one Metallic texture but with specular set to a highgloss. No extra date needed :)

Reply:
Yeah, that's all good, but I meant that `METALLIC_2` looks like the same texture as `METALLIC_1`, just with altered brightness, yet it has its own corresponding texture file. I was wondering if it might make sense to add support for this brightness change with a new material property instead? Maybe just a brightness or opacity factor could make sense?

Reply:
Oh! i am following you; Absolutely if you could git me a way to tweak brightness that would be a HUGE improvement to so many textures